include/openenclave/bits/sgx/sgxtypes.h

```diff
@@ -185,7 +186,10 @@ typedef struct _sgx_sigstruct
     uint32_t miscmask;
 
     /* (908) Reserved. Must be 0. */
-    uint8_t reserved2[20];
+    uint8_t reserved2[4];
```

Comment:
Has it been validated that SGX1 devices that don't support KSS accept sigstructs where the reserved fields are non-zero?